com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/graph.py

```diff
@@ -109,7 +109,10 @@ def addOperator(self, kind, function=None, name=None, params=None, sl=None):
 
         if name is None:
             name = self._requested_name(None,action="Op", func = function)
-
+        
```

Comment:
```
SEVERE: t/parallel_test.spl:63:1: CDISP0025E ERROR: The following identifier is a duplicate: map_lambda_2.
Jun 30, 2017 11:30:07 AM com.ibm.streamsx.topology.internal.process.ProcessOutputToLogger run
SEVERE: t/parallel_test.spl:52:1: CDISP0032D DETAILS: The map_lambda_2 identifier was previously defined at this location.
Jun 30, 2017 11:30:07 AM com.ibm.streamsx.topology.internal.process.ProcessOutputToLogger run
SEVERE:
Jun 30, 2017 11:30:07 AM com.ibm.streamsx.topology.internal.process.ProcessOutputToLogger run
SEVERE: CDISP0092E ERROR: Because of previous compilation errors, the compile process cannot continue.
```